### ISSUE TITLE
Add to_csv() method to dask/dataframe/io.py

### ISSUE THREAD

**Issue by rea725** [ContinuumIO/dask#455]
Wondering if to_csv() pandas wrapper could be added, much like what was already done with read_csv().  We'd be happy to test it :)

**Comment by mrocklin** [comment]
Would the desired output be a single CSV file or a collection of CSV files?  If a collection then what is the best way to specify the naming scheme?

**Comment by rea725** [comment]
I think the default should be a single file, just to replicate pandas directly.  I hadn't contemplated the collection output, but could see its applicability.  I'm not sure I can recommend a preferred way to specify a naming scheme, other than to find some other similar naming convention from another package with significant adoption.

**Comment by rea725** [comment]
Actually,  I think maybe an optional chunksize argument could be specified, and if the output exceeds the chunksize, just use an auto-increment.

**Comment by mrocklin** [comment]
Fixed in the simple one file case in https://github.com/ContinuumIO/dask/pull/458 .

**Comment by mrocklin** [comment]
For chunking we do something similar already in dask.bag.  The user either provides a list of filenames, or a filename with a `*` in it and a function that expects an increment and we use that to generate text that gets inserted into the `*` placeholder.

There is already a natural place at which to chunk (dataframes are themselves chunked).

I hope to wait on writing to collections of csvs until we try out the single csv case.

**Comment by shoyer** [comment]
I would something sort of similar to this for saving netCDFs in xray:
http://xray.readthedocs.org/en/stable/generated/xray.save_mfdataset.html#xray.save_mfdataset
